pipeline/src/lib/apply.svelte

```diff
@@ -1,20 +1,75 @@
-<form class="w-[80%] max-w-6xl mx-auto px-4 py-8">
+<script>
+  export let id;
+  let loading = false;
+  let name = '';
+  let email = '';
+  let interest = '';
+  let skills = '';
+  let workHours = '';
+  let question = '';
+
+  const handleApplySubmit = async () => {
```

Comment:
Ok for now, but in future we'll refactor to form actions:
https://kit.svelte.dev/docs/form-actions